src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -510,219 +1197,133 @@ private Task InvokeAllExceptionFiltersAsync()
                 throw new InvalidOperationException(message);
             }
 
-            await InvokeActionFilterAsync();
+            await InvokeNextActionFilterAsync();
 
             Debug.Assert(_actionExecutedContext != null);
             return _actionExecutedContext;
         }
 
-        private async Task InvokeActionFilterAsync()
+        private async Task InvokeActionMethodAsync()
         {
```

Comment:
This is up to date as of when I updated the PR. If this is missing then I'll address it when I rebase.